src/main/histogram.c

```diff
@@ -320,44 +320,3 @@ histogram_print_info(const histogram * h, FILE * out_file)
 	}
 }
 
-#define BUCKETS_PER_LINE 16
```

Comment:
Did this get moved or removed?

Reply:
Removed, that was just used for debugging the histogram while I was writing it, but I can put it back if you think it would be useful to have. It just didn't conform to the standard Ronen wanted for how the histogram is displayed